sqlglot/dialects/hive.py

```diff
@@ -429,7 +436,8 @@ class Generator(generator.Generator):
             if e.args.get("allow_null")
             else "NOT NULL",
             exp.VarMap: var_map_sql,
-            exp.Create: create_with_partitions_sql,
+            #exp.Create: create_with_partitions_sql,
+            exp.Create: _create_sql,
```

Comment:
This won't be needed if `_create_sql` becomes a `Generator` method.

Reply:
To override `create_sql` in Hive Generator class, the execution flow will go 
```
hive.py:create_sql() -> return create_with_partitions_sql ->
dialect.py:create_with_partitions_sql -> self.create_sql() ->
hive.py:create_sql() -> ...
```

Reply:
Hm I see, makes sense. Can probably leave it as is then